tests/test_piffle.py

```diff
@@ -56,3 +56,13 @@ def test_format(self):
         with pytest.raises(Exception):
             img.format('bogus')
 
+    def test_url_parse(self):
+        img = iiif.IIIFImageClient()
+        # simple API parameters
+        img.parse_from_url('http://imgserver.co/loris/img1/full/full/0/default.jpg')
```

Comment:
You're not actually testing anything except that the method doesn't error, right?  Please add checks to confirm that the `img` object is initialized with the correct values in both cases, add checks for malformed or invalid urls (raises expected error, or whatever), and also test for info.json url if you decide to support that (or as an error case).